discovery/gossiper.go

```diff
@@ -2057,10 +2057,17 @@ func (d *AuthenticatedGossiper) processZombieUpdate(
 	// come through again.
 	baseScid := lnwire.NewShortChanIDFromInt(chanInfo.ChannelID)
 	err = d.cfg.Router.MarkEdgeLive(baseScid)
-	if err != nil {
+	switch {
+	case errors.Is(err, channeldb.ErrNotZombieEdge):
```

Comment:
why don't we return the error?

Reply:
I think there will always be a chance that this could throw an error due to the _other_ channel update coming in while this update is being processed and then the other update could have called `MarkEdgeLive` before this one. In that case, we still want the `fallthrough` in `handleChannelUpdate` to happen. 

(added the missing `return nil` though)